tests/e2e/system_test/test_cluster_full_and_recovery.py

```diff
@@ -0,0 +1,348 @@
+        )
+        pod_blk1_obj.run_io(
+            storage_type="fs",
+            size="1G",
+            io_direction="write",
+            runtime=60,
+        )
+        pod_blk1_obj.get_fio_results()
+        log.info(f"IO finished on pod {pod_blk1_obj.name}")
+        pod_blk1_obj.md5 = cal_md5sum(
+            pod_obj=pod_blk1_obj,
+            file_name="fio-rand-write",
+            block=False,
+        )
+
+        log.info("Create PVC1 CEPH-FS, Run FIO and get checksum")
+        pvc_obj_fs1 = pvc_factory(
+            interface=constants.CEPHFILESYSTEM,
+            project=self.project_obj,
+            size=2,
+            status=constants.STATUS_BOUND,
+        )
+        pod_fs1_obj = pod_factory(
+            interface=constants.CEPHFILESYSTEM,
+            pvc=pvc_obj_fs1,
+            status=constants.STATUS_RUNNING,
+        )
+        pod_fs1_obj.run_io(
+            storage_type="fs",
+            size="1G",
+            io_direction="write",
+            runtime=60,
+        )
+        pod_fs1_obj.get_fio_results()
+        log.info(f"IO finished on pod {pod_fs1_obj.name}")
+        pod_fs1_obj.md5 = cal_md5sum(
+            pod_obj=pod_fs1_obj,
+            file_name="fio-rand-write",
+            block=False,
+        )
+
+        log.info(
+            "Fill the cluster to “Full ratio” (usually 85%) with benchmark-operator"
+        )
+        size = get_file_size(92)
+        self.benchmark_obj = BenchmarkOperatorFIO()
+        self.benchmark_obj.setup_benchmark_fio(total_size=size)
+        self.benchmark_obj.run_fio_benchmark_operator(is_completed=False)
+        self.banchmark_operator_teardown = True
+
+        log.info("Verify used capacity bigger than 85%")
+        sample = TimeoutSampler(
+            timeout=1800,
+            sleep=40,
+            func=self.verify_used_capacity_greater_than_expected,
+            expected_used_capacity=85.0,
+        )
+        if not sample.wait_for_func_status(result=True):
+            log.error("The after 18000 seconds the used capacity smaller than 85%")
+            raise TimeoutExpiredError
+
+        log.info(
+            "Verify Alerts are seen 'CephClusterCriticallyFull' and 'CephOSDNearFull'"
+        )
+        log.info("Verify used capacity bigger than 85%")
+        expected_alerts = ["CephClusterCriticallyFull", "CephOSDNearFull"]
+        sample = TimeoutSampler(
+            timeout=600,
+            sleep=50,
+            func=self.verify_alerts_via_prometheus,
+            expected_alerts=expected_alerts,
+        )
+        if not sample.wait_for_func_status(result=True):
+            log.error(f"The alerts {expected_alerts} do not exist after 600 sec")
+            raise TimeoutExpiredError
+
+        for pod_name in ("mon", "mgr", "osd"):
+            log.info(f"Respin pod {pod_name}")
+            disruption = disruption_helpers.Disruptions()
+            disruption.set_resource(resource=f"{pod_name}")
+            disruption.delete_resource()
+
```

Comment:
Please also verify that the above disrupted ceph pods reaching running state after pod respin

Reply:
done